psi/pmt_test.go

```diff
@@ -33,6 +33,18 @@ import (
 	"github.com/Comcast/gots/packet"
 )
 
+func parseHexString(h string) *packet.Packet {
```

Comment:
I don't like how this function is declared in three separate places, but that's something that can be addressed later.